sale_order_line_date/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale order line date
+====================
+
+This module adds requested date to a sales order lines and propagate it to
+stock moves and procurements.
+When the requested date of the whole sale order is modified the requested date
+of the lines change to match.
+
+Usage
+=====
+
+Create a Quotation or a Sales Order and fill the requested date in the sale
+order line
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/213/8.0
```

Comment:
Repo id is improper: https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt#L81